astropy/nddata/utils.py

```diff
@@ -633,11 +633,11 @@ def __init__(self, data, position, shape=None, side_length=None, wcs=None,
         self.input_position_original = position
         self.shape_input = shape
 
```

Comment:
@larrybradley - Are these changes ok with you? I've just flipped the ordering to `((ymin, ymax), (xmin, xmax))` to be consistent with #3909. Let me know if you have any concerns.